### DISCUSSION TITLE
How to use multiple `height` in multiple breakpoints when using `fullWidth` layout?

### DISCUSSION THREAD

**Discussion by lamualfa** [ascorbic/unpic-img#244] [Q&A] [answered]
# Descriptions

- I don't know about the image dimensions (`width` & `height`).
- I want to give a space for the image. The space has a fixed height & flexible width limited by `max-width`.

# On Desktop Screen
<img width="505" alt="image" src="https://github.com/ascorbic/unpic-img/assets/39755201/d2c79b64-7e05-4dd6-9e1e-dc7bc01eb4c7">

The image fits perfectly with the box when using the desktop screen.

# On Mobile Screen
<img width="349" alt="image" src="https://github.com/ascorbic/unpic-img/assets/39755201/19a3c504-1231-47b0-81bc-826788dc2ce9">

But it's too big for the mobile screen.

# The Code

```tsx
import { Image } from '@unpic/react'

function Header(props) {
  return (
    <header className={cs(props.rootClass, rootClass)}>
      <Link href='/' className={rootLogoClass}>
        <Image
          src='https://img.logoipsum.com/286.svg'
          alt='Logo'
          layout='fullWidth'
          height={50}
        />
      </Link>
    </header>
  )
}
```

**Comment by ascorbic** [comment]
Hi. The component doesn't currently have support for art direction. Your best bet there is to use CSS rules to set the height breakpoints

**Reply by ascorbic** [reply, marked as answer]
The latest release does now support art direction. See this post for details. https://dev.to/ascorbic/level-up-your-web-images-with-unpic-picture-tags-4jh1
